docs/semgrep-code/findings.md

```diff
@@ -145,12 +145,13 @@ To **add notes** to the activity history of a finding:
 
 ## Data retention
 
-The retention period for findings is **5 years**.
+Semgrep, Inc. retains findings data as long as an account remains active. We automatically securely destroy data within **90 days of contract termination** for our **Enterprise** customers.
```

Comment:
```suggestion
Semgrep, Inc. retains findings data as long as an account remains active. We automatically securely destroy data within **90 days of contract termination** for our **Enterprise** customers.
```

I'm not which should stay (automatically or securely) but I think there are too many adverbs in this sentence

Reply:
Also, am I opening up a whole 'nother can of worms by asking what defines an active account?

Reply:
Hi @werrett ! Katie has a good point about "active account" - I don't think we have an inactive state?

Reply:
Hey both, we don't have an inactive state. So today we basically don't delete data but I think we want to retain the ability to do that. So 'active' is a good, intentionally vague, answer :)

Reply:
Got it. Thank you @werrett!

Reply:
I also removed the 'automatically' bit, as its not currently true.